src/cr/cube/matrix/measure.py

```diff
@@ -1856,6 +1902,29 @@ def _stddev_func(self):
         )
 
 
+class _SummedCountMargin(_BaseMarginal):
+    """The 'summed-count-margin' (often called the rows-margin/columns-margin)"""
+
+    @lazyproperty
+    def blocks(self):
+        """List of the 2 1D ndarray "blocks" of the summed count margin.
+
+        These are the base-values and the subtotals.
+        """
+        return [
+            self._apply_along_orientation(np.sum, counts) for counts in self._counts
+        ]
+
+    @lazyproperty
+    def is_defined(self):
+        """True if counts are defined."""
+        try:
+            self._counts
+            return True
+        except ValueError:
+            return False
+
```

Comment:
i don't see any usage of this property, what's used for?

Reply:
It's used in `_assemble_marginal`, to make the marginals optional. I'm not sure, maybe it should just propagate the error instead?

Reply:
I can see further down the line making a `.rows_margin_available` property exposed in `_Slice` though

Reply:
we don't have any integration test that steps in the except branch. This guard seems a little bit fishy to me. Also, when the self._counts raise a `ValueError` ? 
The _counts are defined as:
```
    @lazyproperty
    def _counts(self):
        """List of 2 ndarrays count blocks, in their original payload order

        The counts come from the count measure blocks that are comparable in the correct
        orientation, and include the base values and the subtotals for the orientation.
        """
        if self.orientation == MO.ROWS:
            # --- Use *column* comparable counts (going across rows) ---
            counts = self._second_order_measures.column_comparable_counts.blocks
            # --- Get base values & *row* subtotals
            return [counts[0][0], counts[1][0]]
        else:
            # --- Use *row* comparable counts (going across columns) ---
            counts = self._second_order_measures.row_comparable_counts.blocks
            # --- Get base values & *column* subtotals
            return [counts[0][0], counts[0][1]]
 ```
 That seems they will be always present, so I can't buy the `is_defined` written this way. We need to understand when, in this case, the `is_defined` is False.

Reply:
I see you wrote a unit test for this property but we don't have any integration test that steps in the except branch, and I think an integration tests that exercise this conditional branch is mandatory.

Also, when the self._counts raise a `ValueError` ? 
The _counts are defined as:
```
    @lazyproperty
    def _counts(self):
        """List of 2 ndarrays count blocks, in their original payload order

        The counts come from the count measure blocks that are comparable in the correct
        orientation, and include the base values and the subtotals for the orientation.
        """
        if self.orientation == MO.ROWS:
            # --- Use *column* comparable counts (going across rows) ---
            counts = self._second_order_measures.column_comparable_counts.blocks
            # --- Get base values & *row* subtotals
            return [counts[0][0], counts[1][0]]
        else:
            # --- Use *row* comparable counts (going across columns) ---
            counts = self._second_order_measures.row_comparable_counts.blocks
            # --- Get base values & *column* subtotals
            return [counts[0][0], counts[0][1]]
 ```
 
 That seems they will be always present, so I can't buy the `is_defined` written this way. We need to understand when, in this case, the `is_defined` is False.
 
 We need a real(ish) fixture that exercise the condition when `is_defined` is False, I'm imagining a guard like:
 ` self._counts[0].size > 0` or something similar, but i can't figure out when the base counts are not present.